app/controllers/application_templates_controller.rb

```diff
@@ -9,6 +9,15 @@ def bus
     render 'show'
   end
 
+  def new
+    # make a new empty template
+    # make a draft for that template and go to its edit page
```

Comment:
I would say you can go ahead and remove these comments. Hopefully it's pretty simple to see what your controller method does just from looking at the code.